com.writerduet.writersolo.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.writerduet.writersolo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: writersolo
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.portal.Background
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --system-talk-name=org.freedesktop.login1
+  - --filesystem=xdg-download
+rename-desktop-file: bitwarden.desktop
+rename-icon: bitwarden
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: writersolo
+    buildsystem: simple
+    build-commands:
+      - ar -x WriterSolo-7.0.374-amd64.deb
+      - rm -f WriterSolo-7.0.374-amd64.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+      - rm -rf "${FLATPAK_DEST}/share/icons/hicolor/1024x1024/"
+
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://media.writerduet.com/solo_app/v7/WriterSolo-7.0.374-amd64.deb
+        sha256: c84d448b4ac35914e21fe9ad03271ff961095be0d34a348d875437f9fd946802
```

Comment:
Add a filename here to drop the version part. Then you won't need to change `ar -x WriterSolo-7.0.374-amd64.deb` for each update.